provisioning/device/src/interfaces.ts

```diff
@@ -53,8 +53,22 @@ export interface X509ProvisioningTransport {
   registerX509(registrationId: string, auth: X509, forceRegistration: boolean, callback: (err?: Error, registrationResult?: X509RegistrationResult, body?: any, result?: any) => void): void;
 }
 
+/**
+ * Public API exposed by the X509 security client object.  This is only useful if you're writing your own security client.
+ */
 export interface X509SecurityClient {
+  /**
+   * return the X509 certificate
```

Comment:
>return [](start = 5, length = 6)

nit: it doesn't really return. it calls the callback with either an error or the certificate.